### ISSUE TITLE
"account not found" fake error

### ISSUE THREAD

**Issue by klepsydra** [suddjian/alpaca-cli#3]
I had my hopes up when I learned of this only CLI trader for Alpaca (is there another that works under /bin/bash ?)

To prove I connected my account just fine, with money ready to invest:
```
 alpaca report

Portfolio:
  symbol     qty      cost     value      profit


Performance:
  US$ 0.00


```
Yet when I try to place a simple limit order to buy:

```
 alpaca buy 1 BIZD --type=limit --time-in-force=gtc --limit-price=17.07


404 - {"code":40410000,"message":"account not found for 662d0fac-5c7f-4835-a246-5749cdbf8070"}

```

I had tried updating/reinstalling, but that made no difference:
`npm install -g alpaca-cli  ## https://www.npmjs.com/package/alpaca-cli `

**Comment by suddjian** [comment]
Yeah, this isn't actively maintained since Alpaca and I parted ways last year. If you figure it out, feel free to open a PR and I'll merge it.
